larrow:1.0.0

Mambouna opened the pull request:
<!--
Thanks for submitting a package! Please read and follow the submission guidelines detailed in the repository's README and check the boxes below. Please name your PR as `name:version` of the submitted package.

If you want to make a PR for something other than a package submission, just delete all this and make a plain PR.
-->

I am submitting
- [x] a new package

<!--
Please add a brief description of your package below and explain why you think it is useful to others. If this is an update, please briefly say what changed.
-->

Description: The package allows easily drawing and styling arrows between labels in the document. It's useful because there is currently no easy in-built way to do so. Base typst does not have arrows and cetz (which larrow uses) is non-trivial for putting drawings into the flow of content. The package resolves those.

<!--
These things need to be checked for a new submission to be merged. If you're just submitting an update, you can delete the following section.
-->

I have read and followed the submission guidelines and, in particular, I
- [x] selected [a name](https://github.com/typst/packages/blob/main/docs/manifest.md#naming-rules) that isn't the most obvious or canonical name for what the package does
- [x] added a [`typst.toml`](https://github.com/typst/packages/blob/main/docs/manifest.md#package-metadata) file with all required keys
- [x] added a [`README.md`](https://github.com/typst/packages/blob/main/docs/documentation.md) with documentation for my package
- [x] have chosen [a license](https://github.com/typst/packages/blob/main/docs/licensing.md) and added a `LICENSE` file or linked one in my `README.md`
- [x] tested my package locally on my system and it worked
- [x] [`exclude`d](https://github.com/typst/packages/blob/main/docs/tips.md#what-to-commit-what-to-exclude) PDFs or README images, if any, but not the LICENSE

elegaanz commented:
Yes, you are correct. This is indeed not very clear in the docs and could be improved.

Mambouna commented:
> Yes, you are correct. This is indeed not very clear in the docs and could be improved.

Thank you for the advice, I've reverted the changes and fixed the toml after a failed test.

elegaanz commented:
Looks good, thank you for this package :)
